lib/govuk_app_config/default_configuration.rb

```diff
@@ -0,0 +1,86 @@
+require "govuk_app_config/govuk_data_sync"
+
+class DefaultConfiguration < SimpleDelegator
+  def initialize(raven_configuration_instance)
+    super(raven_configuration_instance)
```

Comment:
The argument here is redundant by the way, `super` is automatically passed a parent methods arguments.